app/models/user.rb

```diff
@@ -3,16 +3,15 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-   #has_many :items
+         has_many :items
 
-  PASSWORD_REGEX = /\A(?=.*?[a-z])(?=.*?\d)[a-z\d]+\z/i.freeze
-  NAME_REGEX = /\A[ぁ-んァ-ン一-龥々]+\z/.freeze
-  NAME_KANA_REGEX = /\A[ァ-ヶー－]+\z/.freeze
-  validates_format_of :password, with: PASSWORD_REGEX, message: 'Include both letters and numbers' 
-    
-  with_options presence: true do
-    validates :nickname, :l_name, :f_name, :l_name_kana, :f_name_kana, :birthday
-    validates :l_name, :f_name, format: { with: NAME_REGEX }
-    validates :l_name_kana, :f_name_kana, format: { with: NAME_KANA_REGEX }
-  end
+         PASSWORD_REGEX = /\A[a-zA-Z0-9]+\z/.freeze
+         NAME_REGEX = /\A[ぁ-んァ-ン一-龥]/.freeze
```

Comment:
```suggestion
         NAME_REGEX = /\A[ぁ-んァ-ン一-龥々]+\z/.freeze
```
こちらの記述ですが、記述が変わっているようなので元に戻しておきましょう。

Reply:
>   NAME_REGEX = /\A[ぁ-んァ-ン一-龥]/.freeze
>   NAME_REGEX = /\A[ぁ-んァ-ン一-龥々]+\z/.freeze
> こちらの記述ですが、記述が変わっているようなので元に戻しておきましょう。

こちらの指摘について、まだ修正が反映されていないようです。ご確認ください。
コミット・プッシュの漏れがないかも併せてご確認いただきますようお願いいたします。